code/modules/surgery/robotics.dm

```diff
@@ -246,6 +246,10 @@
 				var/obj/item/organ/internal/brain/sponge = target.internal_organs_by_name[BP_BRAIN]
 				if(sponge && istype(I, sponge))
 					target.cure_all_traumas()
+				if(istype(tool, /obj/item/stack/nanopaste))
+					var/obj/item/stack/nanopaste/nanopaste = tool
+					nanopaste.use(1)
```

Comment:
Is there a check somewhere to ensure that the nanopaste isnt empty already?

Reply:
No. I'll fix.

Reply:
This isn't addressed. Do this in the `can_use` proc above.
```dm
	if (istype(tool, /obj/item/stack/nanopaste))
		var/obj/item/stack/nanopaste/NP = tool
		if (NP.uses_charge < 1)
			to_chat(user, span("notice", "[NP] is empty and cannot be used for this surgery."))
			return SURGERY_FAILURE
```

Reply:
Not needed. Nanopaste cannot be empty. Even 'empty' sprite nanopaste still has 1 charge left. Upon being used up, the item deletes itself. There is no need to check for if it has charges left unless there's another purpose to it.